### PR TITLE
Update core to version 25955fc5

### PR DESCRIPTION
**Pull request by hculea**

### CODE REVIEW

File: src/release/templates/version.tpl.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # TODO: Refactor so SDK_BUILD_NUMBER can be inside and not hardcoded in defaults.py
```
Comment (hculea) [inline src/release/templates/version.tpl.py:1]: Have we updated `defaults.py`?

Comment (hculea) [reply src/release/templates/version.tpl.py:1]: Have we run the prep-release script at all, to do the version bumping?

Comment (MOmarMiraj) [reply src/release/templates/version.tpl.py:1]: Wouldn't we run the `prep-release` script after we merge?

File: src/release/RELEASE-NOTES [hunk 2]
```diff
@@ -1,7 +1,10 @@
-This is the v0 release of the 1Password Python SDK.
+This is the v0.1.2 release of the 1Password Python SDK.
```
Comment (hculea) [inline src/release/RELEASE-NOTES:1]: note for reviewer - ignore the changes in this file, we're deciding about the content here separately.

**Comment by hculea** [comment]
Functionally tested - everything looks great!

**Review by AndyTitu — APPROVED** [review]

**Pull request merged**